fdbrpc/TokenSign.cpp

```diff
@@ -117,6 +117,71 @@ bool verifyToken(SignedAuthTokenRef signedToken, StringRef publicKeyDer) {
 	return true;
 }
 
+StringRef hmacSHA256Sign(Arena& arena, StringRef str, StringRef key) {
```

Comment:
HMAC is not a digital signature algorithm, it's a message authentication code, where you mix the generic "shared secret key" (basically any byte string shared between message sender and receiver). Therefore it doesn't have to be ASN.1-encoded private key like you'd use in PKI. Just pointing out that the terminology used here is misleading.